src/main/java/network/brightspots/rcv/TabulatorSession.java

```diff
@@ -111,9 +111,10 @@ void convertToCdf() {
   // function: tabulate
   // purpose: run tabulation
   // returns: list of winners
-  void tabulate() throws TabulationCancelledException {
+  void tabulate() {
     Logger.log(Level.INFO, "Starting tabulation session...");
     ContestConfig config = ContestConfig.loadContestConfig(configPath);
+    Boolean tabulationSuccess = false;
```

Comment:
boolean

Reply:
Why this?  I remember we decided this at some point, but I think we should at least consider using Booleans in RawContestConfig.  It allows us to distinguish between missing (null value, or no key) vs. false values in Jackson.

Reply:
Sure, I'm open to Booleans (and other object equivalents for primitives) in RawContestConfig if we don't decide to use all Strings.

But in a case like this, there's no reason not to use a primitive; all you want to store is false or true.